File: Problem 3/sol.cpp
string timeConversion(string s) {
    std::string totalHora = s.substr(0, 8);
    std::string hora = s.substr(0, 2);
    std::string totalMenosHora = s.substr(2, 6);
    std::string em = s.substr(8, 2);
    
    std::string newHora = hora;
    
    if(em == "PM"){
        if(hora!="12"){    
            int entero = std::stoi(hora);
            entero = entero + 12;
            
            newHora = std::to_string(entero);
        }
    }else{
        if(hora=="12"){
            newHora="00";
        }
    }
    return newHora+totalMenosHora;
}

File: Problem 3/sol.js
function timeConversion(s) {
    let hora = s.substring(0,2);
    let totalMenosHora = s.substring(2,8);
    let em = s.substring(8,10);
    
    if(em == "PM"){
        if(hora!="12"){
            let intHora = parseInt(hora);
            intHora += 12;
            hora = intHora.toString();
        }
    }else if(hora=="12"){
        hora = "00";
    }
    
    return hora + totalMenosHora;
}

